### ISSUE TITLE
Crop Stomp just stops working.

### ISSUE THREAD

**Issue by Firerise9402** [RHeavenStudio/HeavenStudio#420]
### Describe the bug

I was using crop stomp before for charting, and those sections still work, but now that I try and chart another part with it, it doesn't work.

### How to reproduce the bug?

Chart using crop stomp and try using the start marching keyframe to start the minigame.

### Log file

```shell
Mono path[0] = 'C:/Users/firer/Downloads/StandaloneWindows64-build (1)/Heaven Studio_Data/Managed'
Mono config path = 'C:/Users/firer/Downloads/StandaloneWindows64-build (1)/MonoBleedingEdge/etc'
Initialize engine version: 2021.3.21f1 (1b156197d683)
[Subsystems] Discovering subsystems at path C:/Users/firer/Downloads/StandaloneWindows64-build (1)/Heaven Studio_Data/UnitySubsystems
GfxDevice: creating device client; threaded=1; jobified=1
Direct3D:
    Version:  Direct3D 11.0 [level 11.1]
    Renderer: NVIDIA GeForce RTX 3060 Ti (ID=0x2489)
    Vendor:   NVIDIA
    VRAM:     8038 MB
    Driver:   31.0.15.2849
Begin MonoManager ReloadAssembly
- Completed reload, in  8.583 seconds
<RI> Initializing input.

<RI> Input initialized.

<RI> Initialized touch support.

UnloadTime: 0.659500 ms
Flushing possible JoyShocks...
No JoyShocks found.
The character used for Underline is not available in font asset [rodin_lat_cy_ja_ko_spec SDF].
Unloading 6 Unused Serialized files (Serialized files now loaded: 0)
UnloadTime: 3.239600 ms
Running game loader RvlBadmintonLoader
Running game loader CtrBearLoader
Running game loader RvlBoardMeetingLoader
Running game loader NtrFlickLoader
Running game loader CtrCatchLoader
Running game loader RvlBookLoader
Running game loader AgbClapLoader
Running game loader NtrCoinLoader
Running game loader NtrCropLoader
Running game loader NtrDjLoader
Running game loader NtrDogNinjaLoader
Running game loader RvlDoubleDateLoader
Running game loader CtrDrummingLoader
Running game loader NtrIdolLoader
Running game loader AgbFireworkLoader
Running game loader CtrFirstContact
Running game loader RvlFlipperFlopLoader
Running game loader RvlForkLoader
Running game loader NtrGleeClubLoader
Running game loader RvlNewKarateLoader
Running game loader CtrTeppanLoader
Running game loader RvlRocketLoader
Running game loader NtrBackbeatLoader
Running game loader AgbMarcherLoader
Running game loader pcoMeatLoader
Running game loader AgbUpbeatLoader
Running game loader ntrMunchyMonkLoader
Running game loader NtrOctopusMachineLoader
Running game loader CtrPillowLoader
Running game loader AgbQuizShowLoader
Running game loader NtrPingpongLoader
Running game loader PcoSomenLoader
Running game loader AgbHairLoader
Running game loader RvlRingsideLoader
Running game loader NtrSamuraiLoader
Running game loader AgbGhostLoader
Running game loader AgbSpaceDanceLoader
Running game loader NtrSoccerLoader
Running game loader AgbBatterLoader
Running game loader RvlTambourineLoader
Running game loader AgbTapLoader
Running game loader RvlTapTroupeLoader
Running game loader NtrDazzlesLoader
Running game loader AgbTramLoader
Running game loader MobTrickLoader
Running game loader NtrTunnelLoader
Running game loader AgbWaltzLoader
Running game loader RvlWorkingDoughLoader
Discord status updated
Create TimingDisplayComponent
Create SkillStarComponent
Create SectionComponent
Unloading 510 unused Assets to reduce memory usage. Loaded Objects now: 6538.
Total: 2.240100 ms (FindLiveObjects: 0.319000 ms CreateObjectMapping: 0.082000 ms MarkObjects: 1.674400 ms  DeleteObjects: 0.164500 ms)

Update Success!
Unknown action mrUpbeat/stepping found in remix.json! Adding action...
Unknown action mrUpbeat/ding! found in remix.json! Adding action...
== Playthrough statistics of Exilelord - Soulless 5 (played at 2023-05-16 12:08:43) ==
Average input offset for playthrough: 0ms
Accuracy for playthrough:  0.00
Cleared 0 sections out of 57
Skill Star not collected
Discord status updated
what
drawing waveform. samples:, 44982272, packsize: 7497
Update Success!
Playing at 1831.5
IndexOutOfRangeException: Index was outside the bounds of the array.
  at HeavenStudio.Datamodel.Split (System.String s, System.Int32 index) [0x00009] in <bb488195bb4843129757ae660201657b>:0 
  at HeavenStudio.Games.CropStomp.Awake () [0x00213] in <bb488195bb4843129757ae660201657b>:0 
UnityEngine.Object:Internal_CloneSingle(Object)
UnityEngine.Object:Instantiate(GameObject)
HeavenStudio.GameManager:SetGame(String)
HeavenStudio.GameManager:SetCurrentEventToClosest(Single)
HeavenStudio.<PlayCo>d__71:MoveNext()
UnityEngine.SetupCoroutine:InvokeMoveNext(IEnumerator, IntPtr)

Section Pirates A started
Section Pirates B started
== Playthrough statistics of Exilelord - Soulless 5 (played at 2023-05-16 12:08:57) ==
Average input offset for playthrough: 0ms
Accuracy for playthrough:  0.00
Cleared 2 sections out of 57
Skill Star not collected
Perfect Clear!
```

### Additional context

By the way don't worry about what I'm currently charting.

**Comment by YumJum90** [comment]
Add another game after crop stomp chart (or only the switch game event if you want to know how's it sounds) that'll work for now as the issue gets fixed

**Comment by Firerise9402** [comment]
> can you send an example of a RIQ with this issue?

I can't seem to reproduce it, and the chart I was working on with the bug is very big, and github won't let me attach RIQ files. I'd have to re-make the bug report to upload one.

**Comment by Firerise9402** [comment]
On the bright side, adding the game after crop stomp does fix it, so there's a temporary fix.

**Comment by minenice55** [comment]
fixed by #463
